docs/using/installing/index.rst

```diff
@@ -340,13 +348,22 @@ The file ``cluster.key`` should be kept only by the cluster administrator; it do
 
 You are now able to generate authentication certificates for the control service and each of your nodes.
 To generate the control service certificate, run the following command from the same directory containing your authority certificate generated in the previous step.
-Replace ``example.org`` with the hostname of your control service node; this hostname should match the one you will give to REST API clients.
+
+If you have a domain name for your cluster, replace ``example.org`` with the hostname of your control service node.
+You can also use an IP address here.
+For TLS verification purposes this hostname must match the one you will give to REST API clients.
 
 .. code-block:: console
 
    $ flocker-ca create-control-certificate example.org
    Created control-example.org.crt. Copy it over to /etc/flocker/control-service.crt on your control service machine and make sure to chmod 0600 it.
    
+Before copying these files you will need to SSH into the node and create a /etc/flocker directory:
+
+.. code-block:: console
+
+   root@mercury:~/$ mkdir -p /etc/flocker
```

Comment:
Generally, we're using a prompt `[root@node]#` for logging into remote nodes.

Also, we are mostly preferring the `prompt` directive over the `code-block` directive.

I'd also add the `chmod` from the subsequent text, to help ensure the security practice is at least partly followed.

so my directive would be:
```rst
.. prompt:: bash [root@node]#
   mkdir -p /etc/flocker
   chmod 700 /etc/flocker
```